### PR TITLE
[Feature](Compaction) Support do full compaction by table id

### PR DESCRIPTION
**Pull request by Yukang-Lian**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 45.28 seconds
 stream load tsv:          505 seconds loaded 74807831229 Bytes, about 141 MB/s
 stream load json:         21 seconds loaded 2358488459 Bytes, about 107 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.3 seconds inserted 10000000 Rows, about 341K ops/s
 storage size: 17167255699 Bytes

**Review by zhannngchen — DISMISSED** [review]
LGTM

**Comment by Yukang-Lian** [comment]
run clickbench

**Comment by zhannngchen** [comment]
pls also update the document `compaction-run.md`

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run clickbench

File: docs/zh-CN/docs/admin-manual/http-actions/be/compaction-run.md [hunk 1]
```diff
@@ -41,8 +42,11 @@ under the License.
 * `tablet_id`
     - tablet的id
 
+* `table_id`
+    - table的id。注意，table_id=xxx只有在compact_type=full时指定才会生效，并且tablet_id和table_id只能指定一个，不能够同时指定，指定table_id后会自动对此table下所有tablet执行full_compaction。
+
 * `compact_type`
-    - 取值为`base`或`cumulative`
+    - 取值为`base`或`cumulative`或`full`。full_compaction的使用场景请参考xxx。
```
Comment (zhannngchen) [inline docs/zh-CN/docs/admin-manual/http-actions/be/compaction-run.md:49]: change to the correct link

File: docs/zh-CN/docs/admin-manual/data-admin/data-recovery.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# 数据恢复
+
+对于Merge on Write表，系统可能会在计算delete bitmap时出现错误，导致重复主键结果出现，此时可以利用full compaction功能进行数据的修复。本功能对于非Merge on Write表无效。
```
Comment (zhannngchen) [inline docs/zh-CN/docs/admin-manual/data-admin/data-recovery.md:29]: Merge on Write表->Unique Key Merge on Write表

Comment (zhannngchen) [inline docs/zh-CN/docs/admin-manual/data-admin/data-recovery.md:29]: "系统可能会在计算delete bitmap时出现错误，导致重复主键结果出现"->"在某些Doris的版本中存在bug，可能会导致系统在计算delete bitmap时出现错误，导致出现重复主键“

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 45.69 seconds
 stream load tsv:          535 seconds loaded 74807831229 Bytes, about 133 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          32 seconds loaded 861443392 Bytes, about 25 MB/s
 insert into select:          29.5 seconds inserted 10000000 Rows, about 338K ops/s
 storage size: 17162186357 Bytes

**Review by zhannngchen — APPROVED** [review]
LGTM

**Review by liaoxin01 — APPROVED** [review]
LGTM

**Pull request merged**